scalafix-tests/src/test/resources/checkSyntax/RemoveXmlLiterals.source

```diff
@@ -130,3 +130,21 @@ import scala.xml.quote._
 object A {
   xml"<a>${1}${2}</a>"
 }
+
+<<< patterns: not supported
```

Comment:
Tests are now written in  plain .scala files, like here https://github.com/scalacenter/scalafix/blob/0308ee38ab3f6dd7b1877746bde5514d0406d1cd/scalafix-tests/input/src/main/scala/test/RemoveXmlLiterals.scala

Can you please rebase on master? It would be good to get this merged before 0.4, which I hope to get out Monday latest.